src/processes/auto-welcome-msg-process.js

```diff
@@ -30,8 +30,12 @@ const automateWelcomeMsgProcess = async (job, done) => {
         const newOwnUser =
           resultPrepopulated.ownUser.username[0].toUpperCase() +
           resultPrepopulated.ownUser.username.slice(1);
+        // const toBeSent = {
+        //   text: `Hi ${newOwnUser}\n\nWelcome to ${process.env.BRAND_NAME}.\nWe’re here to help you navigate the app. If you have any questions, ideas or criticism, feel free to message us.\nMessages will be replied by a mix of AI and human review, so we promise to get you a satisfying answer. Every ideas or feedback will be reviewed by our Product team!\nIn our mission to build a better, healthier social internet, we’re here to help, and to learn from you!\n\nAny questions to start with?`
+        // };
+
         const toBeSent = {
-          text: `Hi ${newOwnUser}\n\nWelcome to ${process.env.BRAND_NAME}.\nWe’re here to help you navigate the app. If you have any questions, ideas or criticism, feel free to message us.\nMessages will be replied by a mix of AI and human review, so we promise to get you a satisfying answer. Every ideas or feedback will be reviewed by our Product team!\nIn our mission to build a better, healthier social internet, we’re here to help, and to learn from you!\n\nAny questions to start with?`
+          text: `Hi ${newOwnUser},\nWelcome to ${process.env.BRAND_NAME}! 🎉\n\nGot questions, ideas, or feedback? Share away! We’re all ears. Your input fuels our mission to make the internet a healthier, happier place.\n\nReady to dive in? Let’s chat! 🚀`
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While the new welcome message is more concise and clear, it's important to ensure that all necessary information is still conveyed. The old message mentioned that responses would be a mix of AI and human review, which set expectations for users about the nature of responses they might receive. If this is still true, consider adding this information back into the new message.
```diff
- text: `Hi ${newOwnUser},\nWelcome to ${process.env.BRAND_NAME}! 🎉\n\nGot questions, ideas, or feedback? Share away! We’re all ears. Your input fuels our mission to make the internet a healthier, happier place.\n\nReady to dive in? Let’s chat! 🚀`
+ text: `Hi ${newOwnUser},\nWelcome to ${process.env.BRAND_NAME}! 🎉\n\nGot questions, ideas, or feedback? Share away! We’re all ears. Your input fuels our mission to make the internet a healthier, happier place. Messages will be replied by a mix of AI and human review.\n\nReady to dive in? Let’s chat! 🚀`
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->